lib/spack/docs/getting_started.rst

```diff
@@ -459,6 +459,31 @@ specification. The operations available to modify the environment are ``set``, `
          prepend_path: # Similar for append|remove_path
            LD_LIBRARY_PATH: /ld/paths/added/by/setvars/sh
 
+Finally, if you intend to cross-compile for a different system architecture,
```

Comment:
@haampie mentions

> Spack doesn't support cross-compilation, so I don't think we should imply with docs that it can.

I think this can be addressed with more-precise language: replace

> if you intend to cross-compile for a different system architecture,

with

> if you want to compile for a different target,